src/main/java/uk/gov/hmcts/cmc/claimstore/models/offers/converters/MadeByEnumConverter.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.cmc.claimstore.models.offers.converters;
+
+import uk.gov.hmcts.cmc.claimstore.models.offers.MadeBy;
+
+import java.beans.PropertyEditorSupport;
+
+public class MadeByEnumConverter extends PropertyEditorSupport {
+
+    @Override
+    public void setAsText(String text) throws IllegalArgumentException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)